src/contexts/account-slice.tsx

```diff
@@ -0,0 +1,102 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { Account, AsyncError } from '../types';
+import { AccountService } from '../services';
+
+const accountService = new AccountService();
+
+export const getAccountDetails = createAsyncThunk(
+  'account/getAccountDetails',
+  async (): Promise<Account | undefined> => {
```

Comment:
define type `Nullable<T> = T | null | undefined` and use that here

Reply:
defined and used it across the application